lib/src/signed_video_tlv.h

```diff
@@ -150,11 +150,25 @@ uint8_t
 read_byte(uint16_t *last_two_bytes, const uint8_t **payload, bool do_emulation_prevention);
 
 /**
- * @brief Decodes the PUBLIC_KEY_TAG from data
+ * @brief Scans the TLV part of a SEI payload and decodes the found tag when it's detected.
  *
- * @returns SVI_OK if decoding was successful, otherwise an error code.
+ * The data is assumed to have been written in a TLV format. tlv_find_tag parses data as long as
```

Comment:
Shouldn't we say what tlv_find_tag_and_decode() does instead of what tlv_find_tag() does not?